src/main/java/study/carsharing/configuration/WebSecurityConfiguration.java

```diff
@@ -40,7 +40,7 @@ protected void configure(HttpSecurity http) throws Exception {
                 .sessionManagement().sessionCreationPolicy(SessionCreationPolicy.STATELESS)
                 .and()
                 .authorizeRequests()
-                .antMatchers("/api/login", "/api/join").permitAll()
```

Comment:
@kid7258 
Spring security prevents all requests default.
So, you should check which page should be open.